File: Ejercicios iOS/Autos-Objective-C/AutoAppp/ViewController.m
//
//  ViewController.m
//  AutoAppp
//
//  Created by Virtualizacion02 on 28/08/23.
//

#import "ViewController.h"

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    [_lableAuto setText:@""];
    
}

- (IBAction)borrar:(id)sender {
    
    [_textMarca setText:@""];
    [_textModelo setText:@""];
    [_lableAuto setText:@""];
    
}

- (IBAction)avanzar:(id)sender {
    
    [_nuevoAuto avanzarWithDistancia:10];
    
    [_lableAuto setText:[_nuevoAuto imprimir]];
    
}

- (IBAction)guardar:(id)sender {
    
    _nuevoAuto=[Auto alloc];
    
    _nuevoAuto=[_nuevoAuto initWithMarca:[_textMarca text] Modelo:[_textModelo text]];
    
    [_lableAuto setText:[_nuevoAuto imprimir]];
    
}
@end

File: Ejercicios iOS/Autos-Swift/AutosSwift/ViewController.swift
//
//  ViewController.swift
//  AutosSwift
//
//  Created by Virtualizacion02 on 28/08/23.
//

import UIKit

class ViewController: UIViewController {

    @IBOutlet weak var lableSalida: UILabel!
    @IBOutlet weak var textModelo: UITextField!
    @IBOutlet weak var textMarca: UITextField!
    var nuevoAuto:Auto!

    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view.
        lableSalida.text = " "
    }

    @IBAction func guardar(_ sender: Any) {
        let marca = textMarca.text ?? ""
        let modelo = textModelo.text ?? ""
        nuevoAuto = Auto(marca: marca, modelo: modelo)
    
        lableSalida.text=nuevoAuto.imprimir()
    }
    @IBAction func borrar(_ sender: Any) {
        textMarca.text = " "
        textModelo.text = " "
        lableSalida.text = " "
    }
    @IBAction func avanzar(_ sender: Any) {
        
        nuevoAuto.avanzarWithDistancia()
        lableSalida.text=nuevoAuto.imprimir()
    }
}